ecms_base/features/custom/ecms_projects/config/install/core.base_field_override.taxonomy_term.project_category.status.yml

```diff
@@ -10,7 +10,7 @@ bundle: project_category
 label: Published
 description: ''
 required: false
-translatable: false
+translatable: true
```

Comment:
This should remain false

Reply:
I thought the status field should be translatable to allow a term's translation to be unpublished.

I also thought the path should be translatable so the actual path reflects the term too? So a path in English to the term 'Book' might look like: `/category/book` and in German it would be: `/kategorie/buch`

Reply:
Ok that makes sense, let's  leave it.